projects/packages/jetpack-mu-wpcom/src/features/verbum-comments/assets/class-verbum-gutenberg-editor.php

```diff
@@ -53,6 +53,13 @@ public function remove_strict_kses_filters() {
 	 * Enqueue the assets for the Gutenberg editor
 	 */
 	public function enqueue_assets() {
+		if (
+			! ( is_singular() && comments_open() )
+			&& ! ( is_front_page() && is_page() && comments_open() )
```

Comment:
@heavyweight, this can become complicated to read. Could you specify when it is not added in the function comment?

Reply:
It's the same check that was originally used in highlander and we ported in the enqueuing of verbum assets but forgot to add for the gunteberg editor assets

Reply:
Ok then! I'm not 100% sure about this if statement, but if it has always worked then it's ok.